src/EditorFeatures/CSharpTest/Interactive/NavigateTo/InteractiveNavigateToTests.cs

```diff
@@ -326,7 +326,7 @@ public void FindOverriddenMembers()
                 var unused = itemDisplay.Glyph;
 
                 Assert.Equal("Name", itemDisplay.Name);
-                Assert.Equal("type DogBed", itemDisplay.AdditionalInformation);
+                Assert.Equal($"{EditorFeaturesResources.Type}DogBed", itemDisplay.AdditionalInformation);
```

Comment:
There are numerous other places this string with the trailing whitespace is used and so far in the PLOC builds it hasn't been an issue, but if you see this being an issue with real LOC then it can be changed easily, especially since we can FAR on the resource string now.